quill-engine/src/main/scala/io/getquill/norm/StablizeLifts.scala

```diff
@@ -5,8 +5,8 @@ import scala.collection.immutable.{ Map => IMap }
 
 private[getquill] object StablizeLifts {
```

Comment:
As this is private, it's probably possible to rename the class. I haven't. But I've renamed the function as an anchor to talk about renaming it.